mix.exs

```diff
@@ -28,7 +28,7 @@ defmodule Nerves.Mixfile do
     [
       {:distillery, "~> 1.4"},
       {:ex_doc, "~> 0.16", only: [:test, :dev]},
-      {:nerves_bootstrap, github: "nerves-project/nerves_bootstrap", only: [:test, :dev]}
+      {:nerves_bootstrap, github: "nerves-project/nerves_bootstrap", branch: "host_tools", only: [:test, :dev]}
```

Comment:
This affects nerves_bootstrap?

Reply:
See https://github.com/nerves-project/nerves_bootstrap/pull/14 for the bootstrap updates.